tools/CheckSignature.ps1

```diff
@@ -15,13 +15,21 @@ Param
     [Parameter(ParameterSetName="GalleryInstall", Mandatory=$true)]
     [switch]$GalleryInstall,
     [Parameter(ParameterSetName="CustomPath", Mandatory=$true)]
-    [string]$CustomPath
+    [string]$CustomPath,
+    [Parameter(ParameterSetName="CurrentDirectory")]
+    [string]$OutputFilePath
 )
 
 function Check-StrongName {
     [CmdletBinding()]
     param([Parameter(ValueFromPipeline=$true)][string]$path)
+    Write-Output "Checking strong name for $path" | Out-File -Append $OutputFilePath
```

Comment:
Do we need a parameter validation to ensure that the `OutputFilePath`  is valid and writable.

Reply:
It's worth checking.
I'll implement it in another PR